packages/terra-section-header/src/terra-dev-site/doc/example/AccordionSectionHeader.jsx

```diff
@@ -1,29 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import SectionHeaderExampleTemplate from 'terra-section-header/lib/terra-dev-site/doc/example/SectionHeaderExampleTemplate';
```

Comment:
Outside the scope of this PR, but I think in the future we should rework these examples to not abstract the section header out of the example. I recently looked at these examples to implement the section header and found them to not be very helpful since you cannot see how to implement the section header itself.